File: programming/c#/Function.cs
using System;

namespace Program
{
    class Program
    {
        static string Info(string name, int age, string gender)
        {
            return $"Your name is {name}.\nYou are {age} years old.\nYou are {gender}.";
        }

        static void Main(string[] args)
        {
            string Charles = Info("Charles", 98, "Male");

            Console.WriteLine("The information of Charles is:\n" + Charles);

            Console.ReadLine();
        }
    }
}

File: programming/c++/Function.cpp
#include <iostream>
#include <string>
#include <sstream>

std::string Info(std::string name, int age, std::string gender)
{
    std::ostringstream str;
    str << "Your name is " << name << ".\nYou are " << age << " years old.\nYou are " << gender << "."; 
    return str.str();
}

int main()
{
    std::cout << Info("Charles", 98, "Male") << std::endl;

    return 0;
}